C/Cbc/build_tarballs.jl

```diff
@@ -0,0 +1,81 @@
+using BinaryBuilder, Pkg
+
+name = "Cbc"
+version = v"2.10.5"
+
+# Collection of sources required to build CbcBuilder
+sources = [
+    GitSource("https://github.com/coin-or/Cbc.git",
+               "7b5ccc016f035f56614c8018b20d700978144e9f"),
+    DirectorySource("./bundled"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/Cbc*
+update_configure_scripts
+
+# Remove misleading libtool files
+rm -f ${prefix}/lib/*.la
+
```

Comment:
Try removing the libtool archives also in `/opt/${target}/${target}`, they're the source of all our problems with libtool, see https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/689:
```suggestion
rm -f /opt/${target}/${target}/lib*/*.la

```